### ISSUE TITLE
[Rule] Not enough whitespace after setting

### ISSUE THREAD

**Issue by bhirsz** [MarketSquare/robotframework-robocop#463]
It could be detected:
```
0401 Robot Framework syntax error: Non-existing setting 'Library StatusLib'. (parsing-error)
```
as
"Not enough whitespace after Library setting" or similar. Same for other settings (if not-existing setting error is catched, we could try to guess possible errors).
